packages/tools/devtools/devtools-view/src/components/data-visualization/TreeHeader.tsx

```diff
@@ -62,6 +69,26 @@ export function TreeHeader(props: TreeHeaderProps): React.ReactElement {
 			>
 				{metadata === undefined ? "" : ` ${metadata}`}
 			</span>
+
+			{sharedTreeSchemaData === undefined ? (
```

Comment:
Check out the `LabelCellLayout` component under `utility-components`. That's designed for this case where you have some label text with an icon next to it, which supports tooltips.

Reply:
The `infoTooltipContent` is type of `React.ReactElement | string`, but as you suggested above, we should think about passing the `VisualTree` for `tooltipContents` right? In this case, should we add a new type to `infoTooltipContent`?

Reply:
```
        export const toolTipContentText = "Visual representation of the data structure of the object.";

	{tooltipContents === undefined ? (
		""
	) : (
		<LabelCellLayout
			icon={<DocumentEdit20Regular />}
			infoTooltipContent={toolTipContentText}
		>
			{tooltipContents}
		</LabelCellLayout>
	)}
```


The code above will have output as picture below, and I don't think this is what we want? 


<img width="2014" alt="Screenshot 2024-03-29 at 08 15 03" src="https://github.com/microsoft/FluidFramework/assets/111468570/247f9ade-e7e5-4633-9904-0537d49a7396">

Reply:
> In this case, should we add a new type to infoTooltipContent?

No, you shouldn't need to. You'll need to transform the `VisualTree` to a react component anyways to render it. So you would just do that conversion, and pass the result to `LabelCellLayout`.

Reply:
And the tooltip rendering should result in a popover, rather than the text being displayed inline. But that should already be the case.

I would also recommend using a different icon from the info one. @kashms any recommendations?

Reply:
FWIW, the icon you were using before is probably better than the info button icon, at least in my opinion. But there is probably a better option long term.

Reply:
As per the conversation with @Josmithr , this will be done in a follow up PR.